src/simplification.jl

```diff
@@ -42,7 +42,7 @@ function struct_ctrb_states(A::AbstractVecOrMat, B::AbstractVecOrMat)
     bitA = A .!= 0
     x = vec(any(B .!= 0, dims=2)) # index vector indicating states that have been affected by input
     for i = 1:size(A, 1) # apply A nx times, similar to controllability matrix
-        x = (bitA * x) .!= 0
+        x = x .| (bitA * x) .!= 0
```

Comment:
Is there a reason this is not `x .= x .| (bitA...` or `x .|= (bitA...`? Wouldn't this do the same thing but not have to reallocate 
 `x` every loop? Might be a very minor increased cost so maybe does not really matter.

Reply:
You're right, we could save some allocations here. A `BitArray` of size `x` is a very minor allocation compared to the arrays allocations that result from the order reduction though :P

Reply:
Perhaps even along the lines of
```suggestion
for k = 1:size(A, 1)
        for i in 1:size(A,1)
            x[i] = any(bitA[i,j] & x[j] for j=1:n)
        end
```
seemed to be quite a bit faster for some medium sized example that I tried.